### ISSUE TITLE
Batch size 1

### ISSUE THREAD

**Issue by matthew-jurewicz** [lucidrains/routing-transformer#12]
Can't find batch normalization in the code, so presumably this works just fine with batch size 1?

**Comment by lucidrains** [comment]
oh hey! so transformers don't use batch norm, they use layer norm, so any batch size is ok! I would do `batch size x gradient accumulation = 32` at least

**Comment by matthew-jurewicz** [comment]
Oh wow, I never knew that!
